Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.yml

```diff
@@ -68,17 +68,21 @@ script:
     - defaultValue: '50'
       description: Limit of results to retrieve.
       name: limit
-    - defaultValue: '1'
-      description: Page number.
+    - description: Page number.
       name: page
+      hidden: true
+    - description: Amount of results per request. Value can be range between 1 and 500. When only page_size is given the first page results will be fetched.
+      name: page_size
+    - description: The URL for the next set of items to return during pagination. (Can be retrieved this URL from a previous call).
```

Comment:
The URL for the next set of items to return during pagination. (This URL can be retrieved from a previous call).